pytensor/link/pytorch/linker.py

```diff
@@ -34,3 +71,6 @@ def create_thunk_inputs(self, storage_map):
             thunk_inputs.append(sinput)
 
         return thunk_inputs
+
+    def record_fn(self, name, fn):
```

Comment:
Is this used?

Reply:
Yup, here https://github.com/pymc-devs/pytensor/pull/1094/files#diff-bc4aa470accf91cb330a5f1f6949b9599d4da6c341290e78e27d684144b41429R64

Reply:
I actually removed this and moved the "conversion" func into the linker, so now the linker is doing all the tracking and stuff. Seemed more reasonable.